site/_includes/partials/toc-under-tree.njk

```diff
@@ -0,0 +1,11 @@
+{#
+  This is used as the TOC on smaller screens (<= 992) that makes use of the white space
+  under the navigation tree when the user scrolls down, in response to:
+  https://github.com/GoogleChrome/developer.chrome.com/issues/3097
+#}
+<div class="toc-container toc-container-under-navigation-tree display-none md:display-block xl:display-none pad-right-400 {% if not tocContents %}visibility-hidden{% endif %}">
```

Comment:
Probably not part of this PR...? 🙂

Reply:
This template keeps reappearing in all PRs from Tokyo. Can you please update / resync / rebase your fork to remove  random stuff commited to your main a while back?